commons/connector/health/health_heartRate.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKQuantityTypeIdentifierHeartRate",
```

Comment:
This can also be combined with other Vital Signs, as categorised here: https://developer.apple.com/documentation/healthkit/
This can be combined with heart rate variability, resting heart rate, etc.